index.html

```diff
@@ -2,22 +2,34 @@
 layout: default
 title: Guides
 ---
+<head>
+  <style>
+    body{
+      background-color: rgba(231, 72, 136, 0.849);
+      font-family: Arial, Helvetica, sans-serif, 'Times New Roman', Times, serif;
+    }
+  </style>
+</head>
 <div class="hero">
   <h2>Rails Girls Guides</h2>
   <p>These guides are built to provide tools and a community for women to understand technology and how to build their ideas. Organize your own events, submit new guides, or just use them to learn Rails. For more, add yourself to the <a href="http://groups.google.com/group/rails-girls-team">team mailing list</a>.</p>
-  <p class="hero-links">
-    <a href="http://railsgirls.com">Learn more</a><br>
-    <a href="http://railsgirls.jp/">Guides in Japanese</a><br>
-    <a href="http://railspremierspas.humancoders.com/">Guides in French</a><br>
-    <a href="https://rgcn.github.io">Guides in Chinese</a><br>
-    <a href="https://rgua.github.io">Guides in Russian</a><br>
-    <a href="https://guides.railsgirls.com/guides-ptbr/">Guides in Brazilian-Portuguese</a><br>
-    <a href="https://railsgirlslima.github.io/">Guides in Spanish</a><br>
-    <a href="https://railsgirls.tw/">Guides in Traditional Chinese</a><br>
-    <a href="https://railsgirlshh.github.io/">Guides in German</a><br>
-    <a href="http://guides.railsgirls.co.il/">Guides in Hebrew</a><br>
-    <a href="http://railsgirls.rug.lviv.ua/">Guides in Ukrainian</a><br>
-  </p>
+  <div class="card" style="width: 18rem;">
+    <div class="card-header">
+      <a href="http://railsgirls.com">Learn more</a>
+    </div>
+    <ul class="list-group list-group-flush">
+      <li class="list-group-item"><a href="http://railsgirls.jp/">Guides in Japanese</a></li>
```

Comment:
I think the list looks better, but these CSS classes also don't exist and don't do anything.